c/zss.c

```diff
@@ -94,6 +96,8 @@ char productVersion[40];
 
 static JsonObject *MVD_SETTINGS = NULL;
 static int traceLevel = 0;
+extern int heartbeat_loop_time;
```

Comment:
This is okay, but we usually use camelCase for variable names everywhere else, so it might be best to follow the style already in the file.